content/rs/administering/database-operations/migrate-to-active-active.md

```diff
@@ -0,0 +1,96 @@
+---
+Title: Migrate a Database to Active-Active
+description:
+weight: $weight
+alwaysopen: false
+categories: ["RS"]
+---
+With [Active-Active databases]({{< relref "/rs/administering/designing-production/active-active.md" >}}), applications can read and write to the same data set from different geographical locations seamlessly and with latency less than 1 ms, without changing the way the application connects to the database.
+Active-Active databases also provide disaster recovery and accelerated data read-access for geographically distributed users.
+
+If you have data in a single-region RS database that you want to migrate to an Active-Active database,
+you'll need to create a new Active-Active database and migrate the data into the new database as a [Replica Of](https://docs.redislabs.com/latest/rs/administering/active-passive/) the existing database.
+This process will gradually populate the data in the Active-Active database.
+
+Before data migration starts, all data is flushed from the Active-Active database.
+The data is migrated to the Active-Active instance where you enabled Replica Of, and then the Active-Active replication copies the data from that instance to the other Active-Active instances.
```

Comment:
What other instances? This only explains about a source single-region database, and the new A/A database.

Reply:
![Image from iOS](https://user-images.githubusercontent.com/50071256/109620228-52138100-7b42-11eb-90d1-880d5a293808.jpg)
Does this explain it? BTW if we can generate such drawing to incorporate in the page that would be great.

Reply:
I will look into creating a graphic. For now, I'm going to aim to reword for clarity.